spec/menu_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'menu'
+require 'order'
+
+describe Menu do
+	describe '#see_menu' do
```

Comment:
I like how you nestle your describe for your methods, it makes reading your unit tests much easier.